### DISCUSSION TITLE
Infinispan 15.0 healthcheck

### DISCUSSION THREAD

**Discussion by exilehanharr** [infinispan/infinispan#12496] [Q&A] [answered]
Hello!

Could you please helm me with fix issue in infinispan Helm Chart?

I used infinispan Helm Chart 0.4.0 to install the server. And I got a lot of warning into logs like:

```
infinispan-0 infinispan 2024-06-10 22:50:25,845 WARN  (non-blocking-thread--p2-t2) [org.infinispan.rest.RestRequestHandler] ISPN012026: The REST invocation [NettyRestRequest{GET /rest/v2/cache-managers/default/health/status, remote=/10.216.68.209:45262, subject=null}] has been deprecated. Please consult the upgrade
```

And I can't find correct endpoint for this check. What should I use here instead?

**Comment by ryanemerson** [comment, marked as answer]
@exilehanharr That's been fixed with https://github.com/infinispan/infinispan-helm-charts/commit/e7ca73f7d4d41e08fb9f09eb0bcccfe2cf4aaa13

It should be available in version 0.4.1 of the chart. @rigazilla can you handle the release?

**Reply by exilehanharr** [reply]
@ryanemerson @rigazilla Thank you! It works.
